subresource-loading.md

```diff
@@ -237,20 +244,38 @@ When returning to the event loop, if there are any chunk IDs on the list to fetc
 - When the fetch returns from the server, map in all fetched chunk IDs and add them to the persistent cache.
 - Then, the original fetch can continue at the top of the algorithm. It should either be served by a mapped-in chunk or 404.
 
+#### Pros
+* Mixing in decelration of the resources in the bundle and scopes covered by it in ways that enable use of both techniques, without requiring hard-to-implement [Cache Digests](https://tools.ietf.org/html/draft-ietf-httpbis-cache-digest-05).
+* Decouples the resource number from the number of chunks in ways that enable higher scalability in cases of closely knit, very small modules.
+* Update flow doesn't require content-addressable URLs, which avoids problems with unnecessary cache invalidation of references, as well as "purge" like semantics when resource version changes.
+  - Unclear if that part would work properly in the "online URL enforcement" revalidation flow.
+
+#### Cons
+* Requires declaring full dependency map on each entry point, that includes all the chunks required by all the entry points. If developers will get that wrong, it can result in spurious downloads.
```

Comment:
The idea is for this manifest to be generated by tools, so I would say s/developers/tools/ here.

Reply:
That's fair.